pkgs/applications/video/gpac/default.nix

```diff
@@ -84,37 +61,13 @@ stdenv.mkDerivation rec {
       "CVE-2023-37765"
       "CVE-2023-37174"
       "CVE-2023-23143"
-      "CVE-2023-5998"
```

Comment:
I think you can remove most of the entries from `knownVulnerabilities`. I went through the listed CVEs and AFAICT most of them have been fixed in 2.4.0:

| NVD Entry                                                         | GPAC Issue                               | GPAC Commit                                                                  |
| ----------------------------------------------------------------- | ---------------------------------------- | ---------------------------------------------------------------------------- |
| [CVE-2023-48958](https://nvd.nist.gov/vuln/detail/CVE-2023-48958) | https://github.com/gpac/gpac/issues/2689 | https://github.com/gpac/gpac/commit/249c9fc18704e6d3cb6a4b173034a41aa570e7e4 |
| [CVE-2023-48090](https://nvd.nist.gov/vuln/detail/CVE-2023-48090) | https://github.com/gpac/gpac/issues/2680 | https://github.com/gpac/gpac/commit/545567420e34f1340aecc719a8be54c9758e89de |
| [CVE-2023-48039](https://nvd.nist.gov/vuln/detail/CVE-2023-48039) | https://github.com/gpac/gpac/issues/2679 | https://github.com/gpac/gpac/commit/73db5ca656ef5b179ee9d7ed399877a018d53730 |
| [CVE-2023-48014](https://nvd.nist.gov/vuln/detail/CVE-2023-48014) | https://github.com/gpac/gpac/issues/2613 | https://github.com/gpac/gpac/commit/66abf0887c89c29a484d9e65e70882794e9e3a1b |
| [CVE-2023-48013](https://nvd.nist.gov/vuln/detail/CVE-2023-48013) | https://github.com/gpac/gpac/issues/2612 | https://github.com/gpac/gpac/commit/cd8a95c1efb8f5bfc950b86c2ef77b4c76f6b893 |
| [CVE-2023-48011](https://nvd.nist.gov/vuln/detail/CVE-2023-48011) | https://github.com/gpac/gpac/issues/2611 | https://github.com/gpac/gpac/commit/c70f49dda4946d6db6aa55588f6a756b76bd84ea |
| [CVE-2023-47465](https://nvd.nist.gov/vuln/detail/CVE-2023-47465) | https://github.com/gpac/gpac/issues/2652 | https://github.com/gpac/gpac/commit/a40a3b7ef7420c8df0a7d9411ab1fc267ca86c49 |
| [CVE-2023-47384](https://nvd.nist.gov/vuln/detail/CVE-2023-47384) | https://github.com/gpac/gpac/issues/2672 | https://github.com/gpac/gpac/commit/75b293cebbaa60cd97cf114121a2c3ff585d24fd |
| [CVE-2023-46932](https://nvd.nist.gov/vuln/detail/CVE-2023-46932) | https://github.com/gpac/gpac/issues/2669 | https://github.com/gpac/gpac/commit/dfdf1681aae2f7b6265e58e97f8461a89825a74b |
| [CVE-2023-46931](https://nvd.nist.gov/vuln/detail/CVE-2023-46931) | https://github.com/gpac/gpac/issues/2664 | https://github.com/gpac/gpac/commit/671976fccc971b3dff8d3dcf6ebd600472ca64bf |
| [CVE-2023-46930](https://nvd.nist.gov/vuln/detail/CVE-2023-46930) | https://github.com/gpac/gpac/issues/2666 | https://github.com/gpac/gpac/commit/3809955065afa3da1ad580012ec43deadbb0f2c8 |
| [CVE-2023-46928](https://nvd.nist.gov/vuln/detail/CVE-2023-46928) | https://github.com/gpac/gpac/issues/2661 | https://github.com/gpac/gpac/commit/0753bf6d867343a80a044bf47a27d0b7accc8bf1 |
| [CVE-2023-46927](https://nvd.nist.gov/vuln/detail/CVE-2023-46927) | https://github.com/gpac/gpac/issues/2657 | https://github.com/gpac/gpac/commit/a7b467b151d9b54badbc4dd71e7a366b7c391817 |
| [CVE-2023-46871](https://nvd.nist.gov/vuln/detail/CVE-2023-46871) | https://github.com/gpac/gpac/issues/2658 | https://github.com/gpac/gpac/commit/03760e34d32e502a0078b20d15ea83ecaf453a5c |
| [CVE-2023-46001](https://nvd.nist.gov/vuln/detail/CVE-2023-46001) | https://github.com/gpac/gpac/issues/2629 | https://github.com/gpac/gpac/commit/e79b0cf7e72404750630bc01340e999f3940dbc4 |
| [CVE-2023-42298](https://nvd.nist.gov/vuln/detail/CVE-2023-42298) | https://github.com/gpac/gpac/issues/2567 | https://github.com/gpac/gpac/commit/d33ecf8142d9103eff9d6fd72fbab4e499e3e9b8 |
| [CVE-2023-41000](https://nvd.nist.gov/vuln/detail/CVE-2023-41000) | https://github.com/gpac/gpac/issues/2550 | https://github.com/gpac/gpac/commit/0018b5e4e07a1465287e7dff69b387929f5a75fa |
| [CVE-2023-39562](https://nvd.nist.gov/vuln/detail/CVE-2023-39562) | https://github.com/gpac/gpac/issues/2537 | https://github.com/gpac/gpac/commit/9024531ee8e6ae8318a8fe0cbb64710d1acc31f6 |
| [CVE-2023-37767](https://nvd.nist.gov/vuln/detail/CVE-2023-37767) | https://github.com/gpac/gpac/issues/2514 | https://github.com/gpac/gpac/commit/d414df635c773b21bbb3a9fbf17b101b1e8ea345 |
| [CVE-2023-37766](https://nvd.nist.gov/vuln/detail/CVE-2023-37766) | https://github.com/gpac/gpac/issues/2516 | https://github.com/gpac/gpac/commit/a64c60ef0983be6db8ab1e4a663e0ce83ff7bf2c |
| [CVE-2023-37765](https://nvd.nist.gov/vuln/detail/CVE-2023-37765) | https://github.com/gpac/gpac/issues/2515 | https://github.com/gpac/gpac/commit/36e1b9900ff638576cb88636bbbe2116ed06dfdc |
| [CVE-2023-37174](https://nvd.nist.gov/vuln/detail/CVE-2023-37174) | https://github.com/gpac/gpac/issues/2505 | https://github.com/gpac/gpac/commit/549ff4484246f2bc4d5fec6760332b43774db483 |
| [CVE-2023-23143](https://nvd.nist.gov/vuln/detail/CVE-2023-23143) | https://github.com/gpac/gpac/issues/2366 | https://github.com/gpac/gpac/commit/af6a5e7a96ee01a139cce6c9e4edfc069aad17a6 |
| [CVE-2023-1452](https://nvd.nist.gov/vuln/detail/CVE-2023-1452)   | https://github.com/gpac/gpac/issues/2386 | https://github.com/gpac/gpac/commit/a5efec8187de02d1f0a412140b0bf030a6747d3f |
| [CVE-2023-1449](https://nvd.nist.gov/vuln/detail/CVE-2023-1449)   | https://github.com/gpac/gpac/issues/2387 | https://github.com/gpac/gpac/commit/8ebbfd61c73d61a2913721a492e5a81fb8d9f9a9 |
| [CVE-2023-1448](https://nvd.nist.gov/vuln/detail/CVE-2023-1448)   | https://github.com/gpac/gpac/issues/2388 | https://github.com/gpac/gpac/commit/8db20cb634a546c536c31caac94e1f74b778b463 |
| [CVE-2023-0841](https://nvd.nist.gov/vuln/detail/CVE-2023-0841)   | Unknown[^1] | Unknown[^1] |
| [CVE-2024-24265](https://nvd.nist.gov/vuln/detail/CVE-2024-24265) | Unknown[^2] | Unknown[^2] |
| [CVE-2024-24266](https://nvd.nist.gov/vuln/detail/CVE-2024-24266) | Unknown[^3] | Unknown[^3] |
| [CVE-2024-24267](https://nvd.nist.gov/vuln/detail/CVE-2024-24267) | https://github.com/gpac/gpac/issues/2571 [^4] | https://github.com/gpac/gpac/commit/d28d9ba45cf4f628a7b2c351849a895e6fcf2234 [^4] |

[^1]: The NVD entry links to a [PoC](https://github.com/qianshuidewajueji/poc/blob/main/gpac/mp3_dmx_process_poc3). [All of the bugs reported by this user have been fixed](https://github.com/gpac/gpac/issues?q=+qianshuidewajueji), but it's not clear if this specific bug is included. It looks like the [Debian](https://security-tracker.debian.org/tracker/CVE-2023-0841) and [Gentoo](https://bugs.gentoo.org/show_bug.cgi?id=CVE-2023-23145#c2) maintainers came to a similar conclusion).
[^2]: The NVD entry links to [this report](https://github.com/yinluming13579/gpac_defects/blob/main/gpac_1.md), but I don't know enough about GPAC to determine if this was fixed upstream.
[^3]: The NVD entry links to [this report](https://github.com/yinluming13579/gpac_defects/blob/main/gpac_2.md), but I don't know enough about GPAC to determine if this was fixed upstream.
[^4]: This wasn't linked in the NVD entry, but it seems to match [the linked report](https://github.com/yinluming13579/gpac_defects/blob/main/gpac_3.md).

Reply:
Thanks a lot for that!